site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc

```diff
@@ -0,0 +1 @@
+An Application's OAuth configuration allows you to specify the logout behavior of an application as either "All applications" or "Redirect only." Specifying the default value of "All applications" will call the registered logout URLs for each application on the Tenant _except_ for the FusionAuth admin application. See the https://github.com/FusionAuth/fusionauth-issues/issues/1699[GitHub issue] for more information.
```

Comment:
Please use dashes in include files, except for the initial underscore:

`site/docs/v1/tech/oauth/_logout-behavior-all-applications.adoc`

Reply:
Also, I've started using a line per sentence, which lets diffs be better.

Reply:
```suggestion
An Application's OAuth configuration allows you to specify the logout behavior of an application as either "All applications" or "Redirect only."
Specifying the default value of "All applications" will call the registered logout URLs for each application on the Tenant _except_ for the FusionAuth admin application.
See the https://github.com/FusionAuth/fusionauth-issues/issues/1699[GitHub issue] for more information.
```